keyboards/bastardkb/charybdis/3x6/kb2040/info.json

```diff
@@ -0,0 +1,11 @@
+{
+    "keyboard_name": "Charybdis (4x6) KB2040",
+    "usb": {
+        "device_version": "2.0.0"
+    },
+    "split": {
+        "soft_serial_pin": "GP0"
```

Comment:
```suggestion
        "enabled": true,
        "soft_serial_pin": "GP0"
        "bootmagic": {
            "matrix": [4, 0]
        }
```

Reply:
@waffle87 is there any way to define the matrix keys for both splits? Will this configuration allow to use both halves key at 4,0 to be used for flashing?

Reply:
you can add a bootmagic section under "split" (eg, `split.bootmagic.matrix`)

Reply:
@drashna so both in `.bootmagic.matrix` for central side and in `.split.bootmagic.matrix` for peripheral side?

Reply:
Yeah, that's what it appears.

Reply:
@drashna thanks. Makes sense now. And given 0,0 is the default value I don't need to add it.